.changeset/empty-nails-push.md

```diff
@@ -0,0 +1,24 @@
+---
+'@openfn/language-common': major
+---
+
+### Key Changes
+
+**`logResponse` function:**
+
+- **Simplified API**: Removed the need to pass `query` parameters as a separate
+  argument
+- **Enhanced functionality**: Now automatically extracts `query` parameters from
+  the response object
+
+**`request` function:**
+
+- **Enhanced response structure**: Now includes `query` parameters in the
```

Comment:
As a general rule we should not be "selling" the changes to the adaptor. Especially for such a small technical change. These are technical release notes, not an advert. Focus on telling users what they need to know when migrating.

On a major version, or a big important minor, you can be a bit more dramatic and promotional. But still your primary audience is exhausted developers who just want to know how to upgrade to this adaptor version (and whether they should bother)

If you want to sell the update and encourage users to migrate, post to community and shout about it (and maybe you can edit a link back into the release notes later)